inji-web/src/components/Common/LanguageSelector.tsx

```diff
@@ -45,12 +44,12 @@ export const LanguageSelector: React.FC = () => {
                 onMouseDown={() => setIsOpen(open => !isOpen)}>
                 <p data-testid={`Language-Selector-Selected-DropDown-${language}`}>{LanguagesSupported.find(lang => lang.value === language)?.label}</p>
                 {isOpen ?
-                    <GradientWrapper>
+                   
                         <RiArrowUpSFill size={20} color={'var(--iw-color-languageArrowIcon)'} />
-                    </GradientWrapper> :
-                    <GradientWrapper>
+                    :
+                    
                         <RiArrowDownSFill size={20} color={'var(--iw-color-languageArrowIcon)'}/>
-                    </GradientWrapper>}
```

Comment:
why is GradientWrapper is removed  ?a

Reply:
added it back again, forget to readd it on the arrow.